examples/lb/loadbalancer/main-datasource.tf

```diff
@@ -0,0 +1,6 @@
+data "vkcs_lb_loadbalancer" "http_balancer" {
+  id = vkcs_lb_loadbalancer.app.id
+  # This is unnecessary in real life.
+  # This is required here to let the example work with loadbalancer resource example. 
+  depends_on = [vkcs_lb_loadbalancer.app]
+}
```

Comment:
lets add an ouput like dbaas one having smth about vip

Reply:
added for allocated vip for app